examples/api/lib/widgets/scrollbar/raw_scrollbar.desktop.0.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for RawScrollbar
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ScrollbarApp());
+
+class ScrollbarApp extends StatelessWidget {
+  const ScrollbarApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
```

Comment:
Can you add this case too? Maybe in a separate sample, or just a bit of `snippet` code instead of a full sample in the docs? This is the easiest way to just turn it off for the whole app, rather than on a case by case basis below.

```suggestion
      scrollBehavior: MaterialScrollBehavior.copyWith(scrollbars: false),
      home: Scaffold(
```

Reply:
Added